src/wmtk/TriMeshOperationExecutor.cpp

```diff
@@ -1,41 +1,33 @@
 
-#include "TriMesh_operations.hpp"
+#include "TriMeshOperationExecutor.hpp"
 
 namespace wmtk {
 
-TriMesh::TriMeshOperationExecutor::PerFaceData TriMesh::TriMeshOperationExecutor::get_per_face_data(
-    const Tuple& t)
+constexpr PrimitiveType PV = PrimitiveType::Vertex;
+constexpr PrimitiveType PE = PrimitiveType::Edge;
+constexpr PrimitiveType PF = PrimitiveType::Face;
+
+TriMesh::TriMeshOperationExecutor::IncidentFaceData
+TriMesh::TriMeshOperationExecutor::get_incident_face_data(const Tuple& t)
 {
-    PerFaceData face_data;
-    face_data.deleted_fid = m_mesh.id(t, PF);
-    Tuple t1_edge = m_mesh.switch_tuple(t, PE);
-    face_data.V_C_id = m_mesh.id(m_mesh.switch_tuple(t1_edge, PV), PV);
-    Tuple t2_edge = m_mesh.switch_tuple(t, PV);
-    t2_edge = m_mesh.switch_tuple(t2_edge, PE);
-
-    face_data.ears[0] = EarGlobalIDs{
+    IncidentFaceData face_data;
+    face_data.fid = m_mesh.id(t, PF);
+    const Tuple t1_edge = m_mesh.switch_tuple(t, PE);
+
+    face_data.opposite_vid = m_mesh.id(m_mesh.switch_tuple(t1_edge, PV), PV);
+    const Tuple t2_edge = m_mesh.switch_tuple(m_mesh.switch_tuple(t, PV), PE);
```

Comment:
it'd be good to have a comment for what's going on this older code - i'm not sure what t1 / t2 are / why switching PV vs PE is right